src/debug/debug.cc

```diff
@@ -4226,6 +4226,9 @@ std::string RecordReplayBasicValueContents(Handle<Object> value) {
 
 // Information in the dependency graph for a JS promise.
 struct PromiseDependencyGraphData {
+  // persistentId of the promise value.
+  int persistentId = 0;
```

Comment:
It would be good to keep the same naming style for this as the other properties.

Reply:
Yeah, there are a lot of Replay specific functions in V8 with inconsistent naming.  Ideally these would all use snake_case to match V8's coding conventions.